agent/test-agent-cli/design/RUNBOOK.md

```diff
@@ -0,0 +1,82 @@
+# Steps to use Test-agent Command Line Interface
+`test-agent-cli` lets you write Testsys test using Bash and help in receiving and sending information from/to the Testsys cluster.
+
+## Prerequisites:
```

Comment:
Since the testsys cli is going to be removed eventually lets switch over to cli instead.

Reply:
I would also remove the step numbers.